eortologio@danchris.github.io/extension.js

```diff
@@ -16,31 +16,27 @@
  * SPDX-License-Identifier: GPL-2.0-or-later
  */
 
-/* exported init */
+import GObject from 'gi://GObject';
+import GLib from 'gi://GLib';
+import St from 'gi://St';
+import Clutter from 'gi://Clutter';
 
-const GETTEXT_DOMAIN = 'eortologio-extension';
+import {Extension, gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
+import * as PanelMenu from 'resource:///org/gnome/shell/ui/panelMenu.js';
+import * as PopupMenu from 'resource:///org/gnome/shell/ui/popupMenu.js';
+import * as Helpers from './helpers.js';
 
-const { GObject, St, Clutter , GLib } = imports.gi;
+import * as Main from 'resource:///org/gnome/shell/ui/main.js';
 
-const ExtensionUtils = imports.misc.extensionUtils;
-const Main = imports.ui.main;
-const PanelMenu = imports.ui.panelMenu;
-const PopupMenu = imports.ui.popupMenu;
-const Me = ExtensionUtils.getCurrentExtension();
-const helpers = Me.imports.helpers;
-
-
-const _ = ExtensionUtils.gettext;
-
-let eortologioPopup;
-let currentDatetime;
+ let eortologioPopup;
```

Comment:
Please, remove unused variable.

Reply:
Resolved with this commit https://github.com/danchris/eortologio-gnome-shell-extension/pull/5/commits/a705e30aeb458bb60aa7805e6c85a3b6bad5e30e